filesystem/util.go

```diff
@@ -26,9 +26,18 @@ func isSameDevice(dev1, dev2 string) (bool, error) {
 
 	var st1, st2 unix.Stat_t
 	if err := Stat(dev1, &st1); err != nil {
+		// Some filesystem such as tmpfs, nfs or etc. does not use block device.
+		// In such case, given device path does not exist,
+		// we regard it is not an error but devices are not same always.
```

Comment:
How about the following comment?

```
Some filesystems like tmpfs and nfs aren't backed by block device files.
In such cases, we should not regard `stat` failure as error. In addition, we can't
judge whether two devices are same or not easily. So let's return `false` here for safety.
```

Reply:
I consider the result of the comparison is just a definition, so using words like "not easy" or "safety" is not suitable, IMO.

Reply:
I revised the comment.